### DISCUSSION TITLE
谁能把绝区中的绳网后面的背景图动画帮我整一份

### DISCUSSION THREAD

**Discussion by nostalgiatan** [share121/inter-knot#1538] [其他]
有了后面的动画做背景绳网肯定能变得更加逼真，有的话踢我一下。

**Comment by gaojiaobei** [comment]
![image](https://github.com/user-attachments/assets/e3befbba-4d19-49b3-bf62-72dc3fa4f8d0)

**Reply by BaCktOZwero** [reply]
做不到，就艾草

**Comment by VacuolePaoo** [comment]
我也需要

**Comment by Xmug** [comment]
![给我也弄一个](https://github.com/user-attachments/assets/e8e595a4-b601-4ce1-9612-f21aa652aff3)
